Detector/DetFCChhHCalTile/compact/FCChh_HCalExtendedBarrel_TileCal.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please also include then that file in `Detector/DetFCChhBaseline1/compact/FCChh_DectMaster.xml` so the overlaps are checked.